src/main/java/net/fuzzycraft/botanichorizons/util/IBlockTooltip.java

```diff
@@ -0,0 +1,12 @@
+package net.fuzzycraft.botanichorizons.util;
+
+import cpw.mods.fml.relauncher.Side;
+import cpw.mods.fml.relauncher.SideOnly;
+import net.minecraft.item.ItemStack;
+
+import java.util.List;
+
+public interface IBlockTooltip {
+    @SideOnly(Side.CLIENT)
+    void addTooltipInformation(ItemStack itemStack, @SuppressWarnings("rawtypes") List tooltipStrings);
```

Comment:
You should enable `enableGenericInjection` (and `enableModernJavaSyntax` while you're at it) in `gradle.properties`. This lets you make these `List<String>` everywhere they are used and remove the suppression notations for all of their associated warnings, which will clean this up a lot. I tested it and there were no issues after enabling them.

```suggestion
    void addTooltipInformation(ItemStack itemStack, List<String> tooltipStrings);
```